app/views/kudos/index.html.erb

```diff
@@ -20,3 +21,23 @@
     <% end %>
   <% end %>
 </div>
+<% end %>
+<% if @kudos %>
+<!--Descriptive page name, messages and instructions-->
+<h2 class="heading"><%= ts('Kudos History') %></h2>
+<!--/descriptions-->
+<!--main content-->
+<% if logged_in? && @kudos.present? %>
```

Comment:
Why do we need to check someone's logged in here - doesn't the controller/routing already do that? Presumably if they're not logged in we don't just want to show them a header with no content, we want to redirect them to log in.